lib/vanagon/engine/pooler.rb

```diff
@@ -36,12 +37,17 @@ def load_token
       end
 
       def token_from_file
-        token_file = File.expand_path("~/.vanagon-token")
+        vanagon_token_file = File.expand_path("~/.vanagon-token")
+        vmfloaty_token_file = File.expand_path("~/.vmfloaty.yml")
 
-        if File.exist?(token_file)
-          return File.open(token_file).read.chomp
+        if File.exist?(vanagon_token_file)
+          puts "Reading vmpooler token from: #{vanagon_token_file}"
+          return File.open(vanagon_token_file).read.chomp
+        elsif File.exist?(vmfloaty_token_file)
+          puts "Reading vmpooler token from: #{vmfloaty_token_file}"
+          conf = YAML.load_file(vmfloaty_token_file)
+          return conf['token']
         end
```

Comment:
Short term, this is probably good. But we should refactor this method a little bit and split each "load a kind of data from a specific source" chunk of code into their own methods.